.github/workflows/build-publish.yml

```diff
@@ -9,6 +9,9 @@ on:
       - master
       - develop
       - 'release/*'
+    paths:
+      - '*/**'
+      - '!contracts/**'
```

Comment:
I would double check this with someone from the core team, AFAIK chainlink packages the contract ABI's as part of a code generation process.